script/upgrades/cCOP/Config.sol

```diff
@@ -58,13 +59,15 @@ library cCOPConfig {
   function cCOPcUSD_PoolConfig(
     Contracts.Cache storage contracts
   ) internal view returns (Config.Pool memory poolConfig) {
-    // TODO: Get the final pool parameters from Roman and update them below.
+    // TODO: Get confirmation from Roman that these are OK
+    // These were taken from the $PUSO pool and adjusted to the COP/USD exchange rate,
+    // which was 0.00023747 at the time of writing
     poolConfig = Config.Pool({
       asset0: contracts.celoRegistry("StableToken"),
       asset1: contracts.deployed("StableTokenCOPProxy"),
       isConstantSum: true,
       spread: FixidityLib.newFixedFraction(3, 1000), // 0.3%, in line with current DT of chainlink feed
-      referenceRateResetFrequency: 5 minutes, // TODO: decide whether to make this 6-7 minutes.
+      referenceRateResetFrequency: 6 minutes,
```

Comment:
@rcroessmann We are increasing this to 6 minutes because chainlink often reports new prices very close to the 5 minute mark, and sometimes the pool becomes inactive for about a minute until we relay the new price to sorted oracles.

Reply:
just realized I had a slight mix up here, what we want to make 6 minutes is the expiry of the report, not the frequency of resetting the bucket sizes. Reverted this here https://github.com/mento-protocol/mento-deployment/pull/226/commits/dab6f821cfa51586228dbbd3fb7931d7328e877e